### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Rxbsxn**
It covers stuff listed in #1576

### CODE REVIEW

**Review by ijlee2 — APPROVED** [review]
@Rxbsxn The PR looks great! I will merge this PR after the CI passes (it takes about 20 minutes, unfortunately).

File: guides/release/routing/specifying-a-routes-model.md [hunk 1]
```diff
@@ -17,11 +17,8 @@ Here's an example of a model hook in use within a route:
 
 ```javascript {data-filename=app/routes/favorite-posts.js}
 import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
 
 export default class FavoritePostsRoute extends Route {
-  @service store;
```
Comment (ijlee2) [inline guides/release/routing/specifying-a-routes-model.md:23]: It looks like a store injection may have been removed from a cut-paste. Will you revert the change in the respective folders?

Comment (Rxbsxn) [reply guides/release/routing/specifying-a-routes-model.md:23]: @ijlee2 Yea, I removed it because its redundant here, we do not use store in that case

Comment (Rxbsxn) [reply guides/release/routing/specifying-a-routes-model.md:23]: @ijlee2 please re-verify

Comment (ijlee2) [reply guides/release/routing/specifying-a-routes-model.md:23]: Ah, I see it now. Thanks for clarifying!

**Review by ijlee2 — APPROVED** [review]
@Rxbsxn Thanks again for helping out with improving the docs. Congratulations on your 1st contribution to the Ember Guides! 🎉
